Makefile

```diff
@@ -0,0 +1,42 @@
+JSONNET_FILES=$(shell find . -type f -not -path './vendor/*' \( -name '*.libsonnet' -o -name '*.jsonnet' \))
+
+.DEFAULT_GOAL := all
+
+.PHONY: check
+check:
+	@echo Checking...
+	@echo $(JSONNET_FILES) | xargs -L1 jsonnetfmt -i
+	@git diff -s --exit-code . || (echo "Build failed: one of more files aren't staged to be committed." && git diff && exit 1)
+
+.PHONY: clean
+clean:
+	@echo Cleaning...
+	@rm -rf environments/kubernetes/manifests
+	@rm -rf environments/openshift/manifests
+	@rm -rf environments/sre/servicemonitors
+	@rm -rf environments/sre/prometheusrules
+	@rm -rf environments/sre/grafana
+
+.PHONY: build
+build:
+	@echo Building...
+	@./build.sh
```

Comment:
If we want this Makefile we should replace the `build.sh` with actual make targets like:

```Makefile
environments/kubernetes/manifests:
jsonnet -J vendor -m environments/kubernetes/manifests environments/kubernetes/main.jsonnet | xargs -I{} sh -c 'cat {} | gojsontoyaml > {}.yaml; rm -f {}' -- {}

# for the rest of the build.sh the same
...
```

Reply:
I'd rather keep the Makefile as a wrapper for more complex scripts. When we have simple commands, like `jb install`, it's OK to have it here, but once it gets more complex, it's easier to maintain shell scripts than make targets.